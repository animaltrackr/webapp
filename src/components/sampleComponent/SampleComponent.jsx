import React, { Component } from 'react';
import PropTypes from 'prop-types';

class SampleComponent extends Component {
	render() {
		return <div>SampleComponent</div>;
	}
}

SampleComponent.propTypes = {};

export default SampleComponent;
