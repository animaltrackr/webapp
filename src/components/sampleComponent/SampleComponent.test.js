import React from 'react';
import { shallow } from 'enzyme';
import SampleComponent from './SampleComponent';

describe('<SampleComponent />', () => {
	test('renders', () => {
		const wrapper = shallow(<SampleComponent />);
		expect(wrapper).toMatchSnapshot();
	});
});
