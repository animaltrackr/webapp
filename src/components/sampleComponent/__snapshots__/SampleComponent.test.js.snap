// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SampleComponent /> renders 1`] = `ShallowWrapper {}`;
